### ISSUE TITLE
Retina version?

### ISSUE THREAD

**Issue by ghost** [fabrizioschiavi/pragmatapro#218]
Fira Code Regular looks too thin on Retina screen, and the bold version is too bold, so they offered a Retina version that's a weight in the middle and works great on hires screens.
Pragmata Pro is fantastic, but I can only use it on my lower-density screens, it's difficult to read on Regular weight on a MacBook Pro and too thick in the Bold version.

Do you by any chance have a "Retina" version in the works?

**Comment by fabrizioschiavi** [comment]
Thanks for this request! I'll add the Retina version

**Comment by doplumi** [comment]
Thanks! If you need a beta-tester, hit me up :)

**Comment by benjamin-rood** [comment]
I run a 4K display on Linux, I'm also interested in a "Retina" weight.

Realted note: I find the Bold weight much too heavy for coding, on any display, I would love something closer to a "semibold" that I can make use of...

**Comment by svipas** [comment]
Yeah "Retina" version would be awesome. I suggest to use font weight `450` for it like "Fira Code" did. And create "Retina" version for all font variants (liga, mono, etc.).

**Comment by stevemolitor** [comment]
Is there a retina version now? I’d prefer a slightly lighter version on my MacOS retina screen. Thx!

**Comment by svipas** [comment]
@stevemolitor I recommend to use this: `defaults write NSGlobalDomain AppleFontSmoothing -int 0` this will disable "font smoothing" and will make your text lighter. More to read here: https://tonsky.me/blog/monitors.

**Comment by benjamin-rood** [comment]
> @stevemolitor I recommend to use this: `defaults write NSGlobalDomain AppleFontSmoothing -int 0` this will disable "font smoothing" and will make your text lighter. More to read here: https://tonsky.me/blog/monitors.

💯  Benas. This is absolutely right. I discovered this when I got a new Mac recently and paired it with multiple 4K displays - the advice in the blog is spot-on in my opinion.

**Comment by svipas** [comment]
@benjamin-rood I just noticed that with Light Theme in IDE it's really harder to see text since letters are very thin, so I'm using `defaults write NSGlobalDomain AppleFontSmoothing -int 1` (1 instead of 0). But with Dark Theme in IDE it looked good, maybe even better since letters weren't so bold (reduced blurry effect). Of course this is totally personal.

**Comment by fabrizioschiavi** [comment]
Thank you all for your valuable input on settings!
I can now contribute to resolving the Retina issue with the [PragmataPro Variable](https://fsd.it/shop/fonts/pragmatapro-variable/) font release:
users can now choose the ideal weight for their coding experience.

![Image](https://github.com/user-attachments/assets/f017441e-1770-45d6-a671-e902071929ff)
